saws/saws.py

```diff
@@ -460,5 +460,5 @@ def run_cli(self):
         print('Version:', __version__)
         print('Theme:', self.theme)
         while True:
-            document = self.aws_cli.run()
+            document = self.aws_cli.run(reset_current_buffer=True)
```

Comment:
Hi @jonathanslenders, what does passing in reset_current_buffer=True do?

If instead of calling run() we use the get_input() method, do we need to make a similar reset_current_buffer=True change?